src/wasm-linker.cpp

```diff
@@ -366,10 +369,30 @@ bool hasI64ResultOrParam(FunctionType* ft) {
   return false;
 }
 
+void Linker::makeDummyFunction() {
+  bool create = false;
+  // Check if there are indirectly called functions
+  for (auto& relocation : out.relocations) {
+    if (relocation->kind == LinkerObject::Relocation::kFunction) {
+      create = true;
+      break;
+    }
+  }
+  if (!create) return;
+  wasm::Builder wasmBuilder(out.wasm);
+  Expression *unreachable = wasmBuilder.makeUnreachable();
+  Function *dummy = wasmBuilder.makeFunction(Name(dummyFunction), std::vector<NameType>(), WasmType::none, std::vector<NameType>(), unreachable);
```

Comment:
I think in c++11 you can just do `{}` instead of those `std::vector<NameType>()`s.